src/hypertable.c

```diff
@@ -1616,3 +1616,62 @@ hypertables_rename_schema_name(const char *old_name, const char *new_name)
 
 	scanner_scan(&scanctx);
 }
+
+typedef struct AccumHypertable
+{
+	List	   *ht_oids;
+	Name		schema_name;
+	Name		table_name;
+} AccumHypertable;
+
+
+static ScanTupleResult
+hypertable_tuple_match_name(TupleInfo *ti, void *data)
+{
+	Oid			relid;
+	FormData_hypertable *form = (FormData_hypertable *) GETSTRUCT(ti->tuple);
+	AccumHypertable *accum = data;
+	Oid			schema_oid = get_namespace_oid(NameStr(form->schema_name), true);
+
+	if (!OidIsValid(schema_oid))
```

Comment:
FWIW, you can do the "filtering" in a separate filter function specified in the scan context. Might be cleaner.

Reply:
I just tried to use the filter function, but in this case it's not much cleaner, because both the `filter` and `tuple_found` functions need to get the `relid` of the table, so there ends up being quite a bit of code duplication.